app/models/foreman_google/google_compute.rb

```diff
@@ -1,15 +1,24 @@
 module ForemanGoogle
   class GoogleCompute
+    attr_reader :identity, :status
+
+    def initialize(service:, zone:, identity: nil)
+      @service = service
+      @zone = zone
+      @identity = identity
+    end
+
     def persisted?
+      !!instance
```

Comment:
This fails with:
```
NameError: undefined local variable or method `instance' for #<ForemanGoogle::GoogleCompute:0x000000001159e848>
Did you mean?  @instance
```

Reply:
And just a question, what `persisted?` means in the context of the VM, I'm not sure if I understand correctly the meaning

Reply:
so it should behave the same way as active record - whether it is saved. So we should (in follow-up prs) allow passing all the attributes to create google compute to this object, but there would still be no VM, in that case it would not be `persisted?`